pkg/gui/presentation/icons/git_icons.go

```diff
@@ -59,3 +61,13 @@ func IconForRemote(remote *models.Remote) string {
 	}
 	return DEFAULT_REMOTE_ICON
 }
+
+func IconForWorktree(worktree *models.Worktree, missing bool) string {
+	if worktree.Main() {
```

Comment:
I would show LINKED_WORKTREE_ICON even for the main worktree, so long as we also show the green asterisk